plugins/MauticCrmBundle/EventListener/LeadSubscriber.php

```diff
@@ -61,43 +61,64 @@ public static function getSubscribedEvents()
      */
     public function onLeadPostSave(Events\LeadEvent $event)
     {
+        $lead = $event->getLead();
+        if ($lead->getEventData('pipedrive.webhook')) {
```

Comment:
Same issue here. Pipedrive specfic code should not be included in generic `EventListener/LeadSubscriber.php`